packages/file_selector/file_selector_ios/pigeons/messages.dart

```diff
@@ -7,10 +7,11 @@ import 'package:pigeon/pigeon.dart';
 @ConfigurePigeon(PigeonOptions(
   dartOut: 'lib/src/messages.g.dart',
   dartTestOut: 'test/test_api.g.dart',
-  objcHeaderOut: 'ios/Classes/messages.g.h',
-  objcSourceOut: 'ios/Classes/messages.g.m',
+  objcHeaderOut: 'ios/file_selector_ios/Sources/file_selector_ios/include/file_selector_ios/messages.g.h',
```

Comment:
[Format linter failed](https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8748673526348583921/+/u/Run_package_tests/format/stdout): 
```suggestion
  objcHeaderOut:
      'ios/file_selector_ios/Sources/file_selector_ios/include/file_selector_ios/messages.g.h',
```